src/main/java/com/microsoft/sqlserver/jdbc/dtv.java

```diff
@@ -1459,6 +1461,7 @@ final void executeOp(DTVExecuteOp op) throws SQLServerException {
                 case NVARCHAR:
                 case LONGNVARCHAR:
                 case NCLOB:
+                case JSON:
```

Comment:
I think we're missing a case in the else (line #1599). This makes me think we're also missing a test case that will go to that path.  Check SQL_VARIANT to see where this is tested.

Reply:
I think we also need a case for JSON in readSqlVariant, and, a test for that